packages/SwingSet/test/test-liveslots.js

```diff
@@ -3,73 +3,17 @@ import test from 'ava';
 import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import { assert, details as X } from '@agoric/assert';
-import { WeakRef, FinalizationRegistry } from '../src/weakref';
 import { waitUntilQuiescent } from '../src/waitUntilQuiescent';
-import { makeLiveSlots } from '../src/kernel/liveSlots';
-
-function capdata(body, slots = []) {
-  return harden({ body, slots });
-}
-
-function capargs(args, slots = []) {
-  return capdata(JSON.stringify(args), slots);
-}
-
-function capdataOneSlot(slot) {
-  return capargs({ '@qclass': 'slot', iface: 'Alleged: export', index: 0 }, [
-    slot,
-  ]);
-}
-
-function capargsOneSlot(slot) {
-  return capargs(
-    [{ '@qclass': 'slot', iface: 'Alleged: export', index: 0 }],
-    [slot],
-  );
-}
-
-function oneResolution(promiseID, rejected, data) {
-  return [[promiseID, rejected, data]];
-}
-
-function buildSyscall() {
```

Comment:
ah... so we had already written `buildSyscall()` but not exported it. Cool.

Reply:
yeah there are a bunch of little helpers in various tests that finally hit my threshold for factoring them out into something common